sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -134,15 +134,15 @@ abstract class DatalabFile {
   }
 }
 
-interface FileManager {
+abstract class BaseFileManager {
```

Comment:
I think you should just call this FileManager. Having every reference need to include "Base" seems like exposing implementation detail. You could keep the FileManager interface and make BaseFileManager implement that, but that seems unnecessary.

Reply:
Calling it `FileManager` would indicate it's a full implementation, this is why I included the `Base` prefix, especially when incorporating the below comment too. I'm moving this to a separate PR so we can continue this discussion there.